src/main.ts

```diff
@@ -29,20 +33,13 @@ const app = new Vue({
     search(q: string) {
       /* See https://github.com/vuejs/vetur/issues/1754#issuecomment-595256501 */
       /* eslint-disable @typescript-eslint/no-explicit-any */
-      const view = this.$root.$children[0].$refs.view as any;
+      const view = this.$root.$children[0].$refs.views as any;
```

Comment:
This should remain `view` because the setting of the corresponding ref in App is still `view`